Sources/BraintreeCore/Analytics/BTAnalyticsService.swift

```diff
@@ -53,9 +53,13 @@ final class BTAnalyticsService: AnalyticsSendable {
     
     /// Sends analytics event to https://api.paypal.com/v1/tracking/batch/events/ via a background task.
     /// - Parameter event: A single `FPTIBatchData.Event`
-    func sendAnalyticsEvent(_ event: FPTIBatchData.Event) {
+    func sendAnalyticsEvent(_ event: FPTIBatchData.Event, sendImmediately: Bool) {
         Task(priority: .background) {
-            await performEventRequest(with: event)
+            if sendImmediately {
+                await sendAnalyticsEvents(with: event)
+            } else {
+                await performEventRequest(with: event)
+            }
```

Comment:
take it or leave it but we should be able to do this:
```suggestion
            sendImmediately ? await sendAnalyticsEvents(with: event) : await performEventRequest(with: event)
```

Reply:
Updated: [c63310c](https://github.com/braintree/braintree_ios/pull/1570/commits/c63310c5bdbd56036205d1d3ddeeae25f7ea8e34)